src/main/scala/edu/berkeley/cs/rise/opaque/Utils.scala

```diff
@@ -458,17 +459,18 @@ object Utils extends Logging {
           isNull)
       case (x: CalendarInterval, CalendarIntervalType) =>
         val months = x.months
+        val days = x.days
```

Comment:
Move these changes to a new PR, as Ankur mentioned?

Reply:
Moved, see #94.